src/deleteAllDocs.js

```diff
@@ -0,0 +1,17 @@
+const { db } = require('./connect')();
+
+db.list()
+.then(res => {
+    let docs = [];
+
+    for (let x of res.rows) {
+        let data = {};
+        data._id = x.id;
+        data._rev = x.value.rev;
+        data._deleted = true;
+
```

Comment:
An even more JS way to do this whole thing is to use a [map function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/map)

```js
const docs = res.rows.map(row => ({
	_id = row.id,
	_rev = x.value.rev,
	_deleted = true
});

db.bulk({ docs }).then(...)
```